lib/rust/ensogl/examples/profiling-run-graph/src/lib.rs

```diff
@@ -194,23 +194,24 @@ fn make_rendering_performance_blocks(
     profile: &Profile<Metadata>,
 ) -> Vec<profiler_flame_graph::Block<Performance>> {
     let mut blocks = Vec::default();
-    let render_stats = profile.metadata().filter_map(|metadata| match metadata.data {
-        Metadata::RenderStats(data) => Some(metadata.as_ref().map(|_| data)),
-        _ => None,
-    });
-    for (prev, current) in render_stats.tuple_windows() {
-        let start = prev.time.into_ms();
-        let end = current.time.into_ms();
-        let row = -1;
-        let label = format!("{:#?}", current.data);
-        let block_type = match current.data.fps {
-            fps if fps > 55.0 => Performance::Good,
-            fps if fps > 25.0 => Performance::Medium,
-            _ => Performance::Bad,
-        };
-        let block = profiler_flame_graph::Block { start, end, row, label, block_type };
-        blocks.push(block);
-    }
+    // FIXME
```

Comment:
This will show FPS data without RenderStats:

```suggestion
fn make_rendering_performance_blocks<M>(
    profile: &Profile<M>,
) -> Vec<profiler_flame_graph::Block<Performance>> {
    profile
        .intervals
        .iter()
        .filter(|interval| matches!(profile[interval.measurement].classify(), Class::OnFrame))
        .map(|interval| interval.interval.start.into_ms())
        .tuple_windows()
        .map(|(start, end)| {
            let label = format!("<frame>");
            let row = -1;
            let block_type = match end - start {
                // 60 FPS
                dt if dt < 17.0 => Performance::Good,
                // 30 FPS
                dt if dt < 34.0 => Performance::Medium,
                // <30 FPS
                _ => Performance::Bad,
            };
            profiler_flame_graph::Block { start, end, row, label, block_type }
        })
        .collect()
}
```

The rest of the RenderStats information was displayed when hovering the mouse over the performance blocks, but we can live without it for now. Some day I could implement periodic snapshots of that info less frequently than every frame.